monitor.py

```diff
@@ -701,3 +702,10 @@ class CredentialReaper(ReaperMonitor):
     TIMESTAMP_FIELD = 'expires'
     MAX_AGE = 1
 ReaperMonitor.REGISTRY.append(CredentialReaper)
+
+class PatronReaper(ReaperMonitor):
+    """Remove patron records that expired more than 30 days ago"""
+    MODEL_CLASS = Patron
+    TIMESTAMP_FIELD = 'authorization_expires'
+    MAX_AGE = 30
```

Comment:
Is this new or did it already exist? I would make it longer in case someone has holds but doesn't get around to renewing their card for a while.

Reply:
That seems fine, but I'm not sure how long people normally take to renew a card.

Reply:
From what I've seen people either renew a card pretty quickly or they wait so long that the ILS has no record of them and they end up getting a new card.

The motivating case for NYPL is that people sign up for temporary cards, don't convert them to full cards, and are now signing up again 6-12 months later because of a big membership push on the library's part. Our ILS has already forgotten about those temporary cards.

Reply:
That's interesting. I'm mostly worried about ODL since the circ manager has the only record of a hold, but 60 days should be enough.

Reply:
If this is a problem we could make it a per-library configuration -- we'd just need to write a reaper that worked differently on a per-library basis.